diff --git a/autogen.sh b/autogen.sh
index 5b4c29f..a850834 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -29,13 +29,13 @@ fi
 
 # Check that gmock is present.  Usually it is already there since the
 # directory is set up as an SVN external.
-if test ! -e gmock; then
-  echo "Google Mock not present.  Fetching gmock-1.7.0 from the web..."
-  curl $curlopts -O https://googlemock.googlecode.com/files/gmock-1.7.0.zip
-  unzip -q gmock-1.7.0.zip
-  rm gmock-1.7.0.zip
-  mv gmock-1.7.0 gmock
-fi
+#if test ! -e gmock; then
+#  echo "Google Mock not present.  Fetching gmock-1.7.0 from the web..."
+#  curl $curlopts -O https://googlemock.googlecode.com/files/gmock-1.7.0.zip
+#  unzip -q gmock-1.7.0.zip
+#  rm gmock-1.7.0.zip
+#  mv gmock-1.7.0 gmock
+#fi
 
 set -ex
 
